src/popup/router/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -29,21 +58,118 @@
 </template>
 
 <script>
+import { mapGetters } from 'vuex';
+import { aettosToAe } from '../../../utils/helper';
 import Modal from '../Modal';
 import Button from '../Button';
-import SignAccountIdenticons from '../SignAccountIdenticons';
-import SignDetailsList from '../SignDetailsList';
+import TransactionOverview from '../TransactionOverview';
+import DetailsItem from '../DetailsItem';
+import TokenAmount from '../TokenAmount';
+import CopyButton from '../CopyButton';
+import Plus from '../../../../icons/plus.svg?vue-component';
+import Minus from '../../../../icons/minus.svg?vue-component';
 import mixin from '../../pages/Popups/mixin';
 
 export default {
   components: {
-    Modal, Button, SignAccountIdenticons, SignDetailsList,
+    Modal,
+    Button,
+    TransactionOverview,
+    DetailsItem,
+    TokenAmount,
+    CopyButton,
+    Plus,
+    Minus,
   },
   mixins: [mixin],
   props: {
     transaction: { type: Object, required: true },
   },
+  data: () => ({ showAdvanced: false }),
+  computed: {
+    ...mapGetters(['getTxSymbol', 'getTxAmountTotal']),
+    code() {
+      return this.transaction.payload || this.transaction.callData;
```

Comment:
I wouldn't name `payload` a `code`, also would be better to decode it than to show something like `ba_Xfbg4g==` (actually, it should be an empty string)